Add steps to configure document and field level security in Serverless (#3950)
elastic/docs-content: deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md

* Learn about [built-in roles](elasticsearch://reference/elasticsearch/roles.md)
* [Define your own roles](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md)
* Learn about the [Kibana](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md) and [Elasticsearch](elasticsearch://reference/elasticsearch/security-privileges.md) privileges you can assign to roles
* Learn how to [control access at the document and field level](/deploy-manage/users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md)

:::{tip}
This topic describes built-in roles in {{stack}} clusters and deployments and explains how to create custom ones. You can also learn about [organization-level](/deploy-manage/users-roles/cloud-organization/user-roles.md#ec_organization_level_roles) roles and [cloud resource access roles](/deploy-manage/users-roles/cloud-organization/user-roles.md#ec_instance_access_roles) in {{serverless-full}}.
To create custom roles for {{serverless-full}}, refer to [](/deploy-manage/users-roles/serverless-custom-roles.md).
:::

:::{admonition} Control access at the document and field level in {{serverless-short}}

> {{serverless-full}} centralizes user management with [Cloud organization-level roles](/deploy-manage/users-roles/cloud-organization/user-roles.md#ec_organization_level_roles). You can configure [document and field-level access control](/deploy-manage/users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md) in {{serverless-full}} projects by creating project-level custom roles assigned to Cloud organization users.

Reviewer: could tighten this up a touch 

```suggestion
{{serverless-full}} centralizes user management with [Cloud organization-level roles](/deploy-manage/users-roles/cloud-organization/user-roles.md#ec_organization_level_roles). You can configure [document and field-level access control](/deploy-manage/users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md) in {{serverless-full}} projects as a part of a project-level custom role.
```
